components/brave_ads/core/internal/serving/prediction/model_based/input_variable/segment/creative_ad_model_based_segment_predictor_input_variable_info.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_SERVING_PREDICTION_MODEL_BASED_INPUT_VARIABLE_SEGMENT_CREATIVE_AD_MODEL_BASED_SEGMENT_PREDICTOR_INPUT_VARIABLE_INFO_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_SERVING_PREDICTION_MODEL_BASED_INPUT_VARIABLE_SEGMENT_CREATIVE_AD_MODEL_BASED_SEGMENT_PREDICTOR_INPUT_VARIABLE_INFO_H_
+
+namespace brave_ads {
+
+struct CreativeAdModelBasedPredictorSegmentInputVariableInfo final {
```

Comment:
It seems like this struct is only used for segment matching variables. Maybe include `Match` in the name somehow? This would better justify the defaults of `value` and `weight` below

Reply:
Changed, and thanks :-)